components/dialog/test/dialog-ifrau.visual-diff.js

```diff
@@ -9,25 +9,62 @@ describe('d2l-dialog-ifrau', () => {
 	let browser, page;
 
 	before(async() => {
-		browser = await puppeteer.launch();
+		browser = await puppeteer.launch({ args: ['--no-sandbox', '--disable-setuid-sandbox'], });
```

Comment:
For heaven's sake. I need this to run locally but sometimes I forget to clear it out of my commits... Sigh.